bindings/python/pymongocrypt/mongocrypt.py

```diff
@@ -698,3 +724,66 @@ def __raise_from_status(self):
         finally:
             lib.mongocrypt_status_destroy(status)
         raise exc
+
+
+class RewrapManyDataKeyOpts(object):
+
+    def __init__(self, provider, master_key=None):
+        """Options given to a `rewrap_many_data_key` operation.
+
+        :Parameters:
+        `provider`: The new KMS provider to use to encrypt the data keys, or ``None`` to use the current KMS provider(s).
```

Comment:
The Parameters should be a rst list. Some of the other methods have this problem as well.

Reply:
Done